### PR TITLE
Versions

### PR DESCRIPTION
**Pull request by barambani**

### CODE REVIEW

File: build.sbt [hunk 1]
```diff
@@ -29,12 +30,19 @@ scalacOptions --= {
...
 }
 
-libraryDependencies ++= Seq(
-  "org.scodec" %% "scodec-core" % "1.11.4",
-  "co.fs2" %% "fs2-core" % "2.0.1",
-  "co.fs2" %% "fs2-io" % "2.0.1" % "test",
-  "org.scalacheck" %% "scalacheck" % "1.14.0" % "test"
-)
+libraryDependencies ++= Def.setting {
+  val fs2Version = CrossVersion.partialVersion(scalaVersion.value) match {
```
Comment (mpilquist) [inline build.sbt]: I'd rather drop 2.11 support

Comment (barambani) [reply build.sbt]: thanks, I will do it 👍

**Comment by barambani** [comment]
I tried to follow your PR here https://github.com/scodec/scodec-bits/pull/140. I'm not sure about what to do with the bincompat, so I disabled the check for the moment. Also I didn't format the code. If you agree I plan to do it in a follow up PR not to mess up the actual changes.

**Comment by mpilquist** [comment]
Thanks! I want to do some more checking of the similar PR in scodec-bits prior to merging but I'll hopefully get to that this weekend. The help is much appreciated!

**Comment by barambani** [comment]
thank you. Happy to help

**Pull request merged**